arch-chroot.in

```diff
@@ -4,9 +4,6 @@ shopt -s extglob
 
 m4_include(common)
 
-setup=chroot_setup
-unshare=0
```

Comment:
Hmm, I think we have a coding style here to do global variable initialization first

See also: [mkinitcpio](https://github.com/archlinux/mkinitcpio/blob/master/mkinitcpio)

Reply:
Also I think the current order makes sense. You'll see var declaration/initialization first, then the code that uses them.

Reply:
Well kind of .... unlike mkinitcpio we have a clunky (m4) include, which makes the final code a complete mess.
Should probably move that instead.